src/wymeditor/editor/base.js

```diff
@@ -2032,18 +2032,15 @@ WYMeditor.editor.prototype.paste = function (str) {
 WYMeditor.editor.prototype.insert = function (html) {
     // Do we have a selection?
     var wym = this,
-        selection = wym.selection(),
+        selection = wym.hasSelection() ? wym.selection() : false,
```

Comment:
Is the ternary operator consistent with wymeditor javascript coding style guidelines? I know we try to avoid it in PolicyStat code, since it doesn't improve code readability, and frequently reduces it. I don't have a strong preference on this.